bindings/examples/erc20_forwarder.rs

```diff
@@ -1,13 +1,11 @@
 use alloy::hex;
-use alloy::primitives::{address, Address, B256, U256};
+use alloy::primitives::{Address, B256, U256, address};
```

Comment:
There seems to be a formatter war going on. Let's address this in a follow-up PR